WikiSentiment/WikiData/Article.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WikiSentiment.DataObjects
+{
+    /// <summary>
+    /// POJO class for a single wikipedia title
+    /// </summary>
+    public class Article
+    {
+        public string ttl { get; set; }
+
+        public int vws { get; set; }
+
+        public Dictionary<string, string> lngl { get; set; }    //linked articles in other languages
+
+        /// <summary>
+        /// Builds an article 
+        /// </summary>
+        /// <param name="_client">http client, wiki API headers required</param>
+        /// <param name="_title">article title</param>
+        /// <param name="_countryCode">two letter language code</param>
+        /// <param name="_views">article views</param>
+        /// <returns></returns>
+        public static async Task<Article> Create(HttpClient _client, string _title, string _countryCode, int _views)
```

Comment:
For a `DataObject` this is a lot of logic. 

Consider an "API Layer" class thing responsible for the actual remote call.
Or not. A lot of things that makes sense in large applications is just needless confusion in small applications.